client/scripts/views/pages/discussions/index.ts

```diff
@@ -258,7 +258,7 @@ const DiscussionsPage: m.Component<
     vnode.state.lookback = {};
     vnode.state.postsDepleted = {};
     vnode.state.topicInitialized = {};
-    vnode.state.topicInitialized[ALL_PROPOSALS_KEY] = true;
+    vnode.state.topicInitialized[ALL_PROPOSALS_KEY] = false;
```

Comment:
Reason for this change?

Reply:
So that `loadNextPage()` gets called the first time it loads the page.
Please check L425

```
if (!vnode.state.topicInitialized[subpage]) {
   app.threads.loadNextPage(options).then((morePostsRemaining) => {
          if (!morePostsRemaining) vnode.state.postsDepleted[subpage] = true;
          m.redraw();
        });
```

Initially the threads were not only fetched from `/bulkThreads` endpoint, but also from `/bulkOffchain` for some reason.
It's seemed redundant to me.

So I've removed the query which fetches all threads `/bulkOffChain` 
Now `/bulkOffChain` only returns the pinned threads (first 4 at the top) and the rest of the threads which are paged based on a `cutoffdate` field are directly loaded from `/bulkThreads`.